src/Provider/UnitTests/Provider.Web/Orchestrators/Part2/SkillOrchestratorTests.cs

```diff
@@ -183,18 +180,15 @@ public async Task WhenCustomDraftSkillsHaveBeenAdded_ShouldBeOrderedByTheirPrefi
 
         private static Vacancy GetTestVacancy()
         {
-            return new Vacancy
-            {
+            return new Vacancy {
```

Comment:
Can we revert these formatting changes please.

Reply:
i think my resharper is the culprit!

Reply:
Stop using resharper

Reply:
Resharper is innocent!
https://marketplace.visualstudio.com/items?itemName=mynkow.FormatdocumentonSave - this is formatting.